devito/interfaces.py

```diff
@@ -302,7 +302,23 @@ def _indices(cls, **kwargs):
 
     def _allocate_memory(self):
         """Allocate memory in terms of numpy ndarrays."""
+        from devito.parameters import configuration
```

Comment:
cyclic imports unfortunately, which didn't seem trivial to fix. So I'll attack them in the future

However, I would expect _allocate_memory to be called only a few times, on startup?